go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/go-logr/logr v1.4.2
 	github.com/google/uuid v1.6.0
 	github.com/jarcoal/httpmock v1.3.1
-	github.com/luthermonson/go-proxmox v0.1.1
+	github.com/luthermonson/go-proxmox v0.2.1
```

Comment:
This should be done in a separate PR. If you really need it the bump, wait until #306 is merged and rebase.

Reply:
I don't mind if the bump is here,
I don't see the value of waiting, 
he just bumped to add the new feature.

Reply:
#306 is merged